en/includes/guides/flows/self-registration.md

```diff
@@ -1,94 +1,118 @@
+**Customize the success message:**
 
-We should now make the sign up button move the user to the next step (SMS OTP) instead of finalizing the registration immediately.
+1. Click on the green **End** node on the canvas.
+2. Edit the title (for example, "Registration Successful!") and the description text to customize the message shown to the user upon completion.
 
-1. Click the **Sign Up** button.
-2. Change its action from **Onboard Password** to **Navigation**.
+**Set the completion properties:**
 
-![Step 4]({{base_path}}/assets/img/guides/flows/flow-registration-step-04.gif){: width="auto" style="display: block; margin: 0;"}
+1. With the **End** node selected, click the gear icon (⚙️) to open the **Flow Completion Properties**.
+2. Configure the desired outcomes:
+   - **Auto Login:** Immediately logs the user in once the flow is completed, without requiring them to enter their credentials again.
 
-### Step 5 – Add an SMS OTP verification step
+![Step 5]({{base_path}}/assets/img/guides/flows/flow-registration-step-05.png){: width="auto" style="display: block; margin: 0;"}
 
-This step verifies the mobile number provided during sign up before completing account creation.
+### Step 6 – Enable the flow
 
-1. Navigate to the **Steps** section and drag and drop an **SMS OTP View** onto the canvas.
-        ![Step 5.1]({{base_path}}/assets/img/guides/flows/flow-registration-step-05-1.gif){: width="auto" style="display: block; margin: 0;"}
+Once the flow has been fully configured and is ready for users:
 
-2. Connect the **Sign Up** button to the **SMS OTP View** and connect the **Verify** button to the green tick.
-        ![Step 5.2]({{base_path}}/assets/img/guides/flows/flow-registration-step-05-2.gif){: width="auto" style="display: block; margin: 0;"}
+1. Click **Save Draft** to save all changes.
+2. On the top-right corner of the Flow Builder, turn the toggle **ON** to publish the flow.
 
-3. Click **Save Draft** to save your changes. When ready, toggle the switch on the top right corner to **Enable** and publish the updated flow.
 
-With these changes, new users will only provide the essentials **Email**, **mobile number**, and **password** and must verify their mobile via SMS OTP before completing registration.
+New users will now go through the simplified registration process with mandatory email verification using a Magic Link.
+
+With these changes, new users will only provide the essentials **Email** and **password** and must verify their email via Magic Link before completing registration.
 
 !!! Note
-        When using a connection based sign up option (such as Google or Microsoft), configure the authorized redirect URL as: `https://accounts.asgardeo.io/t/{tenant-domain}/accounts/register`
+    When using a connection based sign up option (such as Google or Microsoft), configure the authorized redirect URL as: `https://{is_host}/t/{tenant-domain}/accounts/register`
+
+## Reference
```

Comment:
The flow completion properties shouldn't come as a reference. Let's make this a step (before enabling a flow step). 

Step x : Configure flow completion properties